test/index.test.js

```diff
@@ -2,19 +2,45 @@ const request = require("supertest");
 const app = require('../app.js');
 const testPerson = require('./testPerson.json')
 
-test("Test the get method", async (done) => {
-    const response = await request(app).get("/1");
-    expect(response.status).toBe(200);
-    expect(response.body.name).toBe('Luke Skywalker');
-    expect(response.body.films.length).toBe(5);
-    expect(response.body.vehicles.length).toBe(2);
-    expect(response.body.starships.length).toBe(2);
-    expect(response.body).toEqual(testPerson);
-    done();
-})
+jest.mock('got');
+
+describe('Positive tests',()=>{
```

Comment:
`describe` is meant to describe the component that you're testing. "Positive tests" is not a help description